pkgs/by-name/ru/rustdesk-flutter/package.nix

```
@@ -41,30 +41,29 @@
 
 in flutter316.buildFlutterApplication rec {
   pname = "rustdesk";
-  version = "1.2.3-unstable-2024-02-11";
+  version = "1.2.3-2";
   src = fetchFromGitHub {
     owner = "rustdesk";
     repo = "rustdesk";
-    rev = "16db977fd81e14af62ec5ac7760a7661a5c24be8";
-    hash = "sha256-k4gNuA/gZ58S0selOn9+K7+s5AQLkpz+DPI84Fuw414=";
+    rev = version;
+    hash = "sha256-aIGaj338eb8/l1NTgBwrLmx5VCpgGoYpx+p/ViCJgjQ=";
   };
 
   strictDeps = true;
 
   # Configure the Flutter/Dart build
   sourceRoot = "${src.name}/flutter";
-  # curl https://raw.githubusercontent.com/rustdesk/rustdesk/16db977fd81e14af62ec5ac7760a7661a5c24be8/flutter/pubspec.lock | yq
+  # curl https://raw.githubusercontent.com/rustdesk/rustdesk/1.2.3-1/flutter/pubspec.lock | yq
```

```suggestion
  # curl https://raw.githubusercontent.com/rustdesk/rustdesk/1.2.3-2/flutter/pubspec.lock | yq
```
Wouldn't this be the correct URL?

---

Thanks :)